data/features/featureList.yaml

```diff
@@ -0,0 +1,891 @@
+  # by default.
+  # Later, it was enabled by default (still beta), eventually
+  # graduating to general availability.
+- name: ExampleFeature3
+  beta:
+    fromVersion: v1.4
+    state: false #
+    enabledFromVersion: v1.5
+  stable:
+    fromVersion: v1.7
+
+featureGateList:
+# Sorted alphabetically
+- name: Accelerators
+  alpha:
+    fromVersion: v1.6
+  deprecated:
+    fromVersion: v1.11
+- name: AdvancedAuditing
+  alpha:
+    fromVersion: v1.7
+  beta:
+    fromVersion: v1.8
+  stable:
+    fromVersion: v1.12
+- name: AffinityInAnnotations
+  alpha:
+    fromVersion: v1.6
+  deprecated:
+    fromVersion: v1.8
+- name: AllowInsecureBackendProxy
+  beta:
+    fromVersion: v1.17
+- name: AllowExtTrafficLocalEndpoints
+  beta:
+    fromVersion: v1.4
+    state: false
+  stable:
+    fromVersion: v1.7
+- name: AnyVolumeDataSource
+  alpha:
+    fromVersion: v1.18
+- name: APIListChunking
+  alpha:
+    fromVersion: v1.8
+  beta:
+    fromVersion: v1.9
+- name: APIPriorityAndFairness
+  alpha:
+    fromVersion: v1.17
+  beta:
+    fromVersion: v1.20
+- name: APIResponseCompression
+  alpha:
+    fromVersion: v1.7
+  beta:
+    fromVersion: v1.16
+- name: APIServerIdentity
+  alpha:
+    fromVersion: v1.20
+- name: AppArmor
+  beta:
+    fromVersion: v1.4
+- name: AttachVolumeLimit
+  alpha:
+    fromVersion: v1.11
+  beta:
+    fromVersion: v1.12
+  stable:
+    fromVersion: v1.17
+- name: BalanceAttachedNodeVolumes
+  alpha:
+    fromVersion: v1.11
+- name: BlockVolume
+  alpha:
+    fromVersion: v1.9
+  beta:
+    fromVersion: v1.13
+  stable:
+    fromVersion: v1.18
+- name: BoundServiceAccountTokenVolume
+  alpha:
+    fromVersion: v1.13
+  beta:
+    fromVersion: v1.21
+- name: ConfigurableFSGroupPolicy
+  alpha:
+    fromVersion: v1.18
+  beta:
+    fromVersion: v1.20
+- name: CPUManager
+  alpha:
+    fromVersion: v1.8
+  beta:
+    fromVersion: v1.10
+- name: CRIContainerLogRotation
+  alpha:
+    fromVersion: v1.10
+  beta:
+    fromVersion: v1.11
+  stable:
+    fromVersion: v1.21
+- name: CronJobControllerV2
+  alpha:
+    fromVersion: v1.20
+  beta:
+    fromVersion: v1.21
+- name: CSIBlockVolume
+  alpha:
+    fromVersion: v1.11
+  beta:
+    fromVersion: v1.14
+  stable:
+    fromVersion: v1.18
+- name: CSIDriverRegistry
+  alpha:
+    fromVersion: v1.12
+  beta:
+    fromVersion: v1.14
+  stable:
+    fromVersion: v1.18
+- name: CSIInlineVolume
+  alpha:
+    fromVersion: v1.15
+  beta:
+    fromVersion: v1.16
+- name: CSIMigration
+  alpha:
+    fromVersion: v1.14
+  beta:
+    fromVersion: v1.17
+- name: CSIMigrationAWS
+  alpha:
+    fromVersion: v1.14
+  beta:
+    fromVersion: v1.17
+    state: false
+- name: CSIMigrationAWSComplete
+  alpha:
+    fromVersion: v1.17
+- name: CSIMigrationAzureDisk
+  alpha:
+    fromVersion: v1.15
+  beta:
+    fromVersion: v1.19
+    state: false
+- name: CSIMigrationAzureDiskComplete
+  alpha:
+    fromVersion: v1.17
+- name: CSIMigrationAzureFile
+  alpha:
```

Comment:
Does the CSIMigrationAzureFile follow the same pattern as other features (is the original table correct)?
From the original table:
Alpha stage, 1.15 - 1.19, then
Beta stage starts with version 1.21

Reply:
The original is incorrect. The alpha stage ends exactly when we release a version that graduates a feature to beta.
There's never a point where a feature is alpha, goes into limbo for a few minor releases, then becomes beta. If the existing table suggests that, it's actually incorrect Markdown (but an easy mistake to make!)

Reply:
OK. That's what I wanted to check.